### ISSUE TITLE
Add console.log on client side

### ISSUE THREAD

**Issue by radare** [frida/frida-gum#8]
Without having to implicitly pass messages, this is important because most of the time you will not have access to the console of the target app.

as a workaround:

```python
import frida
import sys

pid = int(sys.argv[1])
function_address = int(sys.argv[2], 16)

process = frida.attach(pid)
script = process.session.create_script("""
        function print(x) {
                send({type:"msg",msg:x});
        }
        print ("Hello World");
        """ % function_address)

def on_message(message, data):
        payload = message['payload']
        try:
                type = payload['type']
                if type == 'msg':
                        print payload['msg']
        except:
                print(message)
                pass

script.on('message', on_message)
script.load()

```

**Comment by oleavr** [comment]
Very good point. On the "client" side (python in this case) the `message.type` field is `send` whenever send() was used, and `error` in case of an unhandled exception. Perhaps a third type could be added, and bindings are expected to handle that one explicitly?

**Comment by radare** [comment]
that would be a good way to do that imho. usually "print" is just for showing some notes to help the user, if you want something more complex you can implement more machinery in the client side to handle other kind of messages

**Comment by oleavr** [comment]
Agreed. So then either we change `console.log()` to send this message, or introduce `print()` in the global scope. I would argue the former is rather useless as it is now.

**Comment by oleavr** [comment]
Implemented by 5b1d82ca4ed3351e1240da2f165f7fb29c5ece6f, and now also supported by frida/frida-python@59107bab70a4c2c820bd405bca83b260ec0eb14d and frida/frida-qml@d6b2c463942fa09cf78831bb4024459b307a26bf.

**Comment by radare** [comment]
Cool! Thanks
